core/src/main/scala/sbtorgpolicies/io/io.scala

```diff
@@ -63,4 +63,47 @@ package object io {
            else File.separator)
     }
   }
+
+  object IO {
+
+    def file(path: String): File = new File(path)
+
+    def url(address: String): URL = new URL(address)
+
+    def readLines(file: File): Iterator[String] =
+      Source.fromFile(file).getLines()
+
+    def readBytes(file: File): Array[Byte] = {
+      val is: InputStream    = new FileInputStream(file)
+      val array: Array[Byte] = Stream.continually(is.read).takeWhile(_ != -1).map(_.toByte).toArray
+      is.close()
+      array
+    }
+
+    def write(file: File, content: String, charset: Charset = Charset.forName("UTF-8")): Unit = {
+      val writer = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(file, false), charset))
+      writer.write(content)
+      writer.close()
+    }
+
+    def relativize(base: File, file: File): Option[String] = {
```

Comment:
Can we use https://github.com/47deg/sbt-org-policies/blob/e7ea1334edc0b12504c852fbac766acdf14a1028/core/src/main/scala/sbtorgpolicies/github/GitHubOps.scala#L64 somehow?